app/models/manageiq/providers/embedded_terraform/automation_manager/scm_credential.rb

```diff
@@ -1,5 +1,5 @@
 class ManageIQ::Providers::EmbeddedTerraform::AutomationManager::ScmCredential < ManageIQ::Providers::EmbeddedTerraform::AutomationManager::Credential
   include ManageIQ::Providers::EmbeddedAutomationManager::ScmCredentialMixin
 
-  FRIENDLY_NAME = "Embedded Terraform SCM Credential".freeze
+  FRIENDLY_NAME = "Embedded Terraform Credential".freeze
```

Comment:
Why remove this?  I think it is helpful to differentiate this from the other terraform credential types.

Reply:
It was simpler removing 'one' word,  then fixing 12 failures, only for one word :-|  ...
will take look tomorrow  ...

Reply:
What failures?  This is just for display I thought

Reply:
might be something silly,  only did not want put too much time on it, at the time ...

Reply:
it uses 'shared_examples'  https://github.com/putmanoj/manageiq-providers-embedded_terraform/blob/add-cloud-creds-support-for-terraform-runner/spec/models/manageiq/providers/embedded_terraform/automation_manager/credential_spec.rb#L24